apps/passport-client/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "incremental": false,
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```

Comment:
what's the implication of changing this?

Reply:
The the problem I hit was the import of sub-paths, e.g. `import { listen } from "@parcnet-js/client-helpers/iframe" }`.

"node" is pretty much deprecated now, according to the [TypeScript docs](https://www.typescriptlang.org/tsconfig/#moduleResolution):
> 'node10' (previously called 'node') for Node.js versions older than v10, which only support CommonJS require. You probably won’t need to use node10 in modern code.